libsolidity/ast/Types.cpp

```diff
@@ -41,12 +41,12 @@
 #include <boost/algorithm/string/predicate.hpp>
 #include <boost/algorithm/string/replace.hpp>
 #include <boost/algorithm/string/split.hpp>
-#include <boost/range/adaptor/sliced.hpp>
 #include <boost/range/algorithm/copy.hpp>
 
 #include <range/v3/view/enumerate.hpp>
 #include <range/v3/view/reverse.hpp>
 #include <range/v3/view/transform.hpp>
+#include <range/v3/view/drop.hpp>
```

Comment:
Just a tiny nitpick: we try to put imports in alphabetical order in each group. This should be above `enumerate.hpp`.

Reply:
OK, 
I used clang-format for this kind of thing, Why you don't use it on this repo?

Reply:
We had a hard time fitting our coding style into clang-format, and changing the coding style was controversial. So here we are :)

Reply:
@cameel 
I pushed last changes